subsys/fs/ext2/ext2_diskops.h

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2023 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __EXT2_DISKOPS_H__
+#define __EXT2_DISKOPS_H__
+
+#include <stdint.h>
+
+#include "ext2_struct.h"
+
+/**
+ * @brief Fetch inode into given buffer.
+ *
+ * @param fs File system data
+ * @param ino Inode number
+ * @param inode Buffer for fetched inode
+ */
```

Comment:
Missing @retval entries for all of these

Reply:
done.